AUTHORS.md

```diff
@@ -0,0 +1,10 @@
+# Authors
+
+This file contains the list of people involved in the development 
+of `querystringsafe_base64` along its history.
+
+* Michał Pawłowski <https://github.com/pawlowskimichal>
```

Comment:
My only active account is now: https://github.com/not-raspberry (I've got 3 of them :P ).